src/LogicalCameraSensor.cc

```diff
@@ -36,9 +36,13 @@ class gz::sensors::LogicalCameraSensorPrivate
   /// \brief node to create publisher
   public: transport::Node node;
 
+  public: transport::Node node_logic;
+
   /// \brief publisher to publish logical camera messages.
   public: transport::Node::Publisher pub;
 
+  public: transport::Node::Publisher pub_logic;
```

Comment:
use camelCase and add doxygen comment:

```suggestion
  /// \brief Publisher to publish logical camera frustum information
  public: transport::Node::Publisher pubLogic;
```

Reply:
Updated!